js/index.js

```diff
@@ -1 +1,87 @@
 // Your code goes here
+
+
+//mouseover-done
+let logoText = document.querySelector('h1');
```

Comment:
I also do not set my variables at the top of the page.  Doing it this way helps keep things organized for me.  I'm sure it is the same for you.